src/amo/components/AddonMoreInfo/index.js

```diff
@@ -301,7 +336,21 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
 }
 
 export const mapStateToProps = (state: AppState, ownProps: Props) => {
-  const { addon, i18n } = ownProps;
+  const { addon, i18n, relatedCategories } = ownProps;
+  const categoriesState = state.categories.categories;
+  const appName = state.api.clientApp;
+
+  let relatedCategoriesObj = relatedCategories || null;
```

Comment:
I think you said you added this to help a test pass, but you shouldn't need it.

Reply:
Yes, I need that because when I am running the test, I am always getting relatedCategories, after passing relatedCategories from tests, as null because in `mapStateToProps` as I initialized it to null. That's why I added this check.

Reply:
That just means you need a different set up in the test. We shouldn't add code that we don't need just to get a test to pass - that's a sign there's something wrong with the test.

Reply:
Ohk! I will change that.